src/components/Table/Table.jsx

```diff
@@ -77,7 +77,7 @@ const propTypes = {
     hasUserViewManagement: PropTypes.bool,
     /** If true removes the "table-layout: fixed" for resizable tables  */
     useAutoTableLayoutForResize: PropTypes.bool,
-    wrapCellText: PropTypes.oneOf(['always', 'never', 'auto']),
+    wrapCellText: PropTypes.oneOf(['always', 'never', 'auto', 'alwaysTruncate']),
```

Comment:
I think it would help if the comment for this prop described what the intention of each prop means.